server/swap/swap.go

```diff
@@ -1386,8 +1386,8 @@ func (s *Swapper) processInit(msg *msgjson.Message, params *msgjson.Init, stepIn
 	if stepInfo.isBaseAsset {
 		reqFeeRate = stepInfo.match.FeeRateBase
 	}
-	if contract.FeeRate() < reqFeeRate {
```

Comment:
A related change I've been meaning to make, probably next day or two, is to skip the fee rate check if the txn turns out to be mined already.   Just a note.

Reply:
Good idea

Reply:
Done in https://github.com/decred/dcrdex/pull/1458